x-pack/plugins/apm/public/application/action_menu/index.tsx

```diff
@@ -15,7 +15,7 @@ import { useApmPluginContext } from '../../context/apm_plugin/use_apm_plugin_con
 import { AlertingPopoverAndFlyout } from './alerting_popover_flyout';
 import { AnomalyDetectionSetupLink } from './anomaly_detection_setup_link';
 
-export function ActionMenu() {
+export function ApmHeaderActionMenu() {
```

Comment:
Renamed this so it is not confused with the other [Action Menu](https://github.com/elastic/kibana/blob/19b8aa798ca2c373e0c96bd47eab9c65bf00c673/x-pack/plugins/observability/public/components/shared/action_menu/index.tsx#L74-L76)

Reply:
In that case, I might be a good idea to rename the folder too.

Reply:
Good point.